### ISSUE TITLE
Quick orders & Product numbers - Dropdown zone and extended configuration cannot be performed by keyboard alone

### ISSUE THREAD

**Issue by yusufttur** [shopware/platform#8628]
### Shopware Version

6.7.0.0

### Affected area / extension

Extension:B2B-Suite

### Actual behaviour

There is no way to perform the function using only the keyboard on the same screen or on a qualifying conforming alternate version.

![Image](https://github.com/user-attachments/assets/9d9dfa83-c2e5-48aa-96d3-dc601845856b)
![Image](https://github.com/user-attachments/assets/a39aac57-a5cd-4dfc-a6c2-f6777cdfb165)

### Expected behaviour

All interactive components must be:

- Reachable and operable using only the keyboard.
- Fully accessible via focus indicators.
- Accessible to screen readers and assistive tech.

### How to reproduce

- Navigate on the quick order page using only a keyboard.
- Attempt to interact with each element using:
- Tab, Shift + Tab to navigate
- Enter, Space, or arrow keys to activate or select

**Observe:**

- Items in this section do not receive focus.
- No visible focus indicator is present.
- No action can be performed without a mouse.
- Fix this issue by ensuring the component can be used by the keyboard.

**How to fix:**

Utilizing standard HTML form controls and link elements guarantees keyboard operation. User agents provide the keyboard operation of standard controls and additionally map them to an accessibility API. The API is then used by assistive technologies to extract the necessary information and relate them to the user. Standard controls and links include:

<a>
<button>
<input>(various types including, but not limited to: "text", "checkbox", "radio")
<select>
<textarea>
If you create a custom version of a native HTML element or a custom control or widget that does not have a native HTML equivalent, you must add keyboard focusability (via tabindex=”0”) and keyboard events via JavaScript as well as the relevant element role(s) using ARIA.
